lib/spoom/cli/commands/bump.rb

```diff
@@ -0,0 +1,61 @@
+# typed: strict
+# frozen_string_literal: true
+
+require 'find'
+require 'open3'
+
+require_relative 'base'
+
+module Spoom
+  module Cli
+    module Commands
+      class Bump < Base
+        extend T::Sig
+
+        default_task :bump
+
+        desc "bump", "change Sorbet sigils from one strictness to another when no errors"
+        option :from, type: :string
+        option :to, type: :string
+        option :force, type: :boolean, default: false, aliases: :f
+        sig { params(directory: String).void }
+        def bump(directory = ".")
+          from = options[:from] ? options[:from] : Sorbet::Sigils::STRICTNESS_FALSE
```

Comment:
Can we use `Sorbet::Sigils::STRICTNESS_FALSE` as the option `:default` value?

Reply:
I get this error when I try: `uninitialized constant Spoom::Sorbet::Sigils (NameError)`